bioblend/galaxy/tool_dependencies/__init__.py

```diff
@@ -12,13 +17,13 @@ def __init__(self, galaxy_instance):
 
     def summarize_toolbox(
         self,
-        index=None,
-        tool_ids=None,
-        resolver_type=None,
-        include_containers=False,
-        container_type=None,
-        index_by="requirements",
-    ):
+        index: Optional[int] = None,
+        tool_ids: Optional[List[str]] = None,
+        resolver_type: Optional[str] = None,
+        include_containers: bool = False,
+        container_type: Optional[str] = None,
+        index_by: str = "requirements",
```

Comment:
If only "requirements" and "tools" are allowed here, you can:
```suggestion
        index_by: Literal["requirements", "tools"] = "requirements",
```

and import `Literal` from the third-party `typing_extensions` library (see https://github.com/galaxyproject/bioblend/pull/426 ).